packages/contracts/src/version.ts

```diff
@@ -0,0 +1 @@
+export const CONTRACTS_LIRARY_VERSION = 'v0.15.2'
```

Comment:
In the case of version drift, we would expect this value to be different from the value returned by `sphinxLibraryVersion()`.